goals/project_priorities.md

```diff
@@ -0,0 +1,39 @@
+1. The production OpenShift cluster will be initially built and tested by the Red Hat Research group and the MOC engineering staff, working together.
+2. The cluster will be physically located in the MGHPCC.  NERC donated server hardware for the initial build is located at Row 7 Pod C Cage 03 for the infra cluster and Row 7 Pod C Cage 03 (controllers) & Row 1 Pod A Cage 08 (workers) for the prod cluster and Row 7 Pod B Cage 19 for the test cluster.  Note that additional hardware in NET2 racks is not part of this deployment.   (see https://docs.google.com/document/d/1wleTt-j-EDp9Ix75mdinhbQqmlK4udMAp9BtJ6faOpY /edit for original proposal for the elastic data center.)
+3. The cluster will use a public repo (https://github.com/OCP-on-NERC) for all docs, configurations, and new software that isn’t already supported in a different public repo.  (Secrets will of course not be stored in a public repo.) The project to build and operate this cluster will be a public project that is closely related to the Operate First community, and submits new code to that community wherever possible. (The Acceptable Use Policy for this project may differ from the general Operate First AUP, depending on BU/Harvard University requirements.)
+4. The cluster will support university researchers (including students), as well as open source software developers.  Use cases will include providing remote computing, storage, and adequate network bandwidth to support all end users.  The exact number of users has not been forecast, but the working estimate is up to 1000. The exact number of projects is not forecast, but the working estimate is 400.   (Scaling will be addressed in the reference architecture document, not with testing at scale.)   All users are remote  and use the public Internet to access cluster resources.  Some users will also need public IP addresses for their applications/projects.  (NERC typically uses a bastion host with firewall rules to provide this.).  A rough estimate from the NERC group is that the final production cluster will need to include about 2000 cores to be adequate for the expected number of users and projects.  We expect initially one physical rack to be allocated for the production cluster in the MGHPCC, but the plan for this deployment includes only the hardware specified in assumption two.  Additional worker nodes will be added in later phases as the cluster grows.
+5. Volume storage for user and operations data will be provided by NESE.  (Ceph Rados Block Device storage has been used successfully in Operate First, but Ceph File System storage has not, so we will start with just Ceph RBD.) Initial allocation for storage will be 700TB in NESE, in addition to any local storage available in the rack.   (An outstanding question exists on how to integrate with NESE, because NESE requires that CephFS is used for NESE managed DTNs because they must share the same IdP.  e.g. sets of DTNs run Globus, MinIO/S3, NET2/xrootd.)

```

Comment:
This was accepted as is by NERC.  The question was what set of DTNs would be used for the OpenShift connections, and whether there would be overlap with other DTNs.  I don't think this will be an issue for the intial delivery, but sounded like it could  possibly be an issue later if we were working with data that was originally associated with a different set of DTNs.